middleware/replacer.go

```diff
@@ -95,6 +95,7 @@ func NewReplacer(r *http.Request, rr *ResponseRecorder, emptyValue string) Repla
 				dir, _ := path.Split(r.URL.Path)
 				return dir
 			}(),
+			"{locale}": r.Header.Get("Detected-Locale"),
```

Comment:
This can already be done with `{>Detected-Locale}`, why make another way to do it?